src/View/pages/animatedGame/components/area/area.tsx

```diff
@@ -0,0 +1,60 @@
+import { observer } from 'mobx-react';
+import * as React from 'react';
+import { ZoneShape } from 'src/zone/zoneShape';
```

Comment:
Maybe core Interfaces should be separated in one place.
It's bad practice that your component get knowledge about zone